documentation/features/scheduling.rst

```diff
@@ -0,0 +1,223 @@
+
+This information goes beyond the usual time series recorded by an asset's sensors. It's being sent through the API when triggering schedule computation.
+Some parts of it can be persisted on the asset & sensor model as attributes (that's design work in progress). 
+
+
+The flex-context
+-----------------
+
+The ``flex-context`` is independent of the type of flexible device which is optimized.
+With the flexibility context, we aim to describe the system in which the flexible assets operates:
+
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 25 90
+
+   * - Field
+     - Example value
+     - Description 
+   * - ``inflexible-device-sensors``
+     - ``[3,4]``
+     - Power sensors that are relevant, but not flexible, such as a sensor recording rooftop solar power connected behind the main meter, whose production falls under the same contract as the flexible device(s) being scheduled.
+   * - ``consumption-price-sensor``
+     - ``5``
+     - The sensor which defines costs/revenues of consuming energy.
+   * - ``production-price-sensor``
+     - ``6``
+     - The sensor which defines cost/revenues of producing energy.
+   * - ``site-power-capacity``
+     - ``"45kW"``
+     - Maximum/minimum achievable power at the grid connection point [#asymmetric]_. Defaults to the Asset attribute ``capacity_in_mw``
+   * - ``site-consumption-capacity``
+     - ``"45kW"``
+     - Maximum consumption power at the grid connection point [#consumption]_. If ``site-power-capacity`` is defined, the minimum between the ``site-power-capacity`` and ``site-consumption-capacity`` will be used. Defaults to the Asset attribute ``consumption_capacity_in_mw``
+   * - ``site-production-capacity``
+     - ``"0kW"``
+     - Maximum production power at the grid connection point [#production]_.  If ``site-power-capacity`` is defined, the minimum between the ``site-power-capacity`` and ``site-production-capacity`` will be used. Defaults to the Asset attribute ``production_capacity_in_mw``
+
+
+.. [#asymmetric] ``site-consumption-capacity`` and ``site-production-capacity`` allow defining asymmetric contracted transport capacities for each direction (i.e. production and consumption).
+.. [#production] Example: with a connection capacity (``site-power-capacity``) of 1 MVA (apparent power) and a production capacity (``site-production-capacity``) of 400 kW (active power), the scheduler will make sure that the grid outflow doesn't exceed 400 kW.
+.. [#consumption] Example: with a connection capacity (``site-power-capacity``) of 1 MVA (apparent power) and a consumption capacity (``site-consumption-capacity``) of 800 kW (active power), the scheduler will make sure that the grid inflow doesn't exceed 800 kW.
+
+.. warning:: If no (symmetric, consumption and production) site capacity is defined (also not as defaults), the scheduler will not enforce any bound on the site power.
```

Comment:
Consider turning this into a *note*, and noting that the flexible device still has its own power limit defined in its flex-model.